docs/xls-33d-multi-purpose-tokens/reference/mpt_holders.md

```diff
@@ -92,8 +92,8 @@ An `mptoken` object has the following parameters:
 |:-----------------------|:--------|:------------------------------------------|
 | `account`              | string  | The account address of the holder who owns the `MPToken`. |
 | `flags`                | number  | The flags assigned to the`MPToken` object. |
-| `mpt_amount`           | string  | Hex-encoded amount of the holder's balance. |
-| `locked_amount`        | string  | Hex-encoded amount of the locked balance. (Can be omitted if the value is 0.) |
+| `mpt_amount`           | string  | Base 10-encoded amount of the holder's balance. |
```

Comment:
Not a blocker, but should we be saying base 10-encoded or decimal/hexadecimal? I don't have strong opinion either way, but I want to standardize the language we use in this instance.

Reply:
If we're going to say "hex" elsewhere, then I think it makes more sense to say "decimal" here.